gulpfile.js

```diff
@@ -132,7 +154,7 @@ gulp.task("test", (done) => {
     });
 });
 
-gulp.task("build", gulp.series("clean", "ts-compile", "html-webpack", "node_modules-webpack"));
+gulp.task("build", gulp.series("clean", "ts-compile", "html-webpack", "node_modules-webpack", /* "insert-serial-monitor-cli" */));

```

Comment:
The task is commented out now because it will break the build until we actually do a serial-monitor-cli release.